src/components/Toggle/index.js

```diff
@@ -33,12 +33,15 @@ class Toggle extends Component {
     const {selectedItem} = this.state
 
     return (
-      <div className='w5 relative flex items-center br-pill ba b--light-navy bg-dark-navy'>
-        <label className={labelClasses} htmlFor={`left-${leftOption.replace(/\s/g, "-")}`}>
+      <div className='w5 relative flex items-center br-pill ba b--dark-gray bg-transparent'>
+        <label 
+          className={`${sharedLabelClasses} ${selectedItem === 'left' ? 'white' : 'dark-gray'}`}
+          htmlFor={`left-${leftOption.replace(/\s/g, "-")}`}
+        >
           {leftOption}
          <input onClick={onClick} type='radio' name={leftOption} id={`left-${leftOption.replace(/\s/g, "-")}`} className='absolute o-0' checked={selectedItem === 'left'} onChange={this.handleChange} />
         </label>
-        <label className={labelClasses}>
+        <label className={`${sharedLabelClasses} ${selectedItem === 'right' ? 'white' : 'dark-gray'}`}>
```

Comment:
Updated this to work on dark and light backgrounds and now the label shows up better on the rounded toggle switch

![image](https://cloud.githubusercontent.com/assets/5497885/24267016/8420bbb0-0fce-11e7-9e43-62edc4950a0f.png)